content/docs/tutorials/building_public_actors.md

```diff
@@ -11,9 +11,9 @@ paths:
 There are four main stages of building a public actor:
 
 1. [Development]({{@link actors/development.md}}).
-2. [Publication]({{@link actors/publishing.md}}).
+2. [Publication](../../academy/apify_platform/publishing_actors_on_apify_store.md).
```

Comment:
Please use the `{@link}`s to link to other files instead of these absolute paths. Otherwise, things can (and will) break. For example, try clicking on **Publication** in the [deployment preview](https://docs.apify.com/tutorials/building-public-actors?version=academy-content-migration-publishing-actors) for this branch. It doesn't work, because the file path is different from the actual URL path that leads to the page you want.

Reply:
I had a hard time using `{@link}` to link files from the docs to the academy. For example, the build was returning an error for this path `({{@link academy/apify_platform/publishing_actors_on_apify_store.md}})`.

Could you please give me an example of how it should be done for this particular link so I can then update the others?

Reply:
You're right. The `build` script doesn't allow for using `../` to access the parent directory within `@link` tags because the **academy** and **docs** are built separately. So, you cannot safely reference docs articles from the academy, or academy lessons from the docs section. Because of this, I'd recommend just using URLs to the content instead of file paths.